media-gfx/yafaray/yafaray-3.3.0.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6} )
+inherit cmake-utils python-r1 python-utils-r1
```

Comment:
Please don't inherit `python-utils-r1` directly, it contains common part of all other eclasses.